Kernel/Arch/aarch64/ASM_wrapper.h

```diff
@@ -64,11 +64,11 @@ inline void wait_cycles(int n)
 {
```

Comment:
For the commit message, I think you mean: Clang doesn't like using a 32-bit variable with a **64-bit** register
implicitly.